### PR TITLE
Resolved issue #13

### PR DESCRIPTION
**Pull request by camaechi**
To raise the camera height, simply press the period key ('.') repeatedly until you reach your desired height. To lower the height, press the comma (',') key repeatedly.

### CODE REVIEW

**Comment by camaechi** [comment]
README file has been updated to reflect changes.

File: src/config.h [hunk 1]
```diff
@@ -2,7 +2,7 @@
 #define _config_h_
 
 // app parameters
-#define DEBUG 0
+#define DEBUG 1
```
Comment (TonePoems) [inline src/config.h:5]: What is the point of this debug flag being checked in here? Is this required for your feature?

File: src/main.c [hunk 2]
```diff
@@ -2291,6 +2295,19 @@ void on_key(GLFWwindow *window, int key, int scancode, int action, int mods) {
...
+    if (key == GLFW_KEY_COMMA) //Lower Height
+    {
+        if (g->p_height > 2)        
+            g->p_height -= 1;            
+    }
+
+    if (key == GLFW_KEY_PERIOD) //Raise Height
+    {
+        State *s = &g->players->state;
+        s->y += 1;
```
Comment (TonePoems) [inline src/main.c:2308]: You do not seem to reset this y height when lowering. Is this intended?

File: src/main.c [hunk 3]
```diff
@@ -2481,13 +2498,13 @@ void handle_movement(double dt) {
         }
         s->x += vx;
         s->y += vy + dy * ut;
-        s->z += vz;
-        if (collide(2, &s->x, &s->y, &s->z)) {
+        s->z += vz;   		// collide - height is the block height or character height
```
Comment (TonePoems) [inline src/main.c:2501]: are these LSEP characters intended? Doesnt look like a line break intended here

**Review by TonePoems — CHANGES_REQUESTED** [review]
Will be interested to see how this looks in the testing PR!

File: src/config.h (hunk 1 above)
Comment (camaechi) [reply src/config.h:5]: no it was changed to increase commit count

File: src/main.c (hunk 2 above)
Comment (camaechi) [reply src/main.c:2308]: With a height lower than 2, you will get stuck on the blocks and won't be able to walk

File: src/main.c (hunk 3 above)
Comment (camaechi) [reply src/main.c:2501]: that block is used to test the collision and movements

**Review by TonePoems — APPROVED** [review]

**Review by dburd08 — APPROVED** [review]
Looks great, I like this feature, I have not seen something like it in the real minecraft game

**Pull request merged**